sktime/tests/_config.py

```diff
@@ -60,6 +60,10 @@
         "test_fit_does_not_overwrite_hyper_params",
         "test_save_estimators_to_file",
     ],
+    "LSTMFCNClassifier": [
```

Comment:
Any minimal tests specific for the class would be great to have!

Reply:
I agree, although I think this should be a new PR on its own, since we can add common basic tests to all DL Classes.